yt_dlp/extractor/playeur.py

```diff
@@ -9,8 +9,8 @@
 )
 
 
-class UtreonIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?utreon\.com/v/(?P<id>[\w-]+)'
+class PlayeurIE(InfoExtractor):
```

Comment:
The class name should not be changed to maintain compatibility with existing archives.
If you want to display a new name while downloading, use `IE_NAME`.

Reply:
So filename also can't be changes?

Reply:
Yeah, I would leave the filename unchanged.